src/sha1.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable */
+/**
+ * Copied from https://github.com/pH200/hmacsha1-js/blob/master/index.js
+ */
+
+/**
+ * Computes a HMAC-SHA1 code.
+ *
+ * @param {string} k Secret key.
+ * @param {string} d Data to be hashed.
+ * @return {string} The hashed string.
+ */
+export function HmacSHA1(k, d, _p = undefined, _z = undefined) {
```

Comment:
The readable boi did not want to work.
Added the export, renamed to function added to =undefined to the _p and _z parameters.
Prettier did the formatting.